docs/release-process.md

```diff
@@ -64,10 +64,11 @@ The timing of the next Stacking cycle can be found [here](https://stx.eco/dao/to
    - Add cherry-picked commits to the `feat/X.Y.Z.A.n-pr_number` branch
    - Merge `feat/X.Y.Z.A.n-pr_number` into `release/X.Y.Z.A.n`.
 
-4. Open a PR to update the [CHANGELOG](../CHANGELOG.md) file in the `release/X.Y.Z.A.n` branch.
+4. Open a PR to update the [CHANGELOG](../CHANGELOG.md) in the `release/X.Y.Z.A.n` branch.
```

Comment:
```suggestion
4. If necessary, open a PR to update the [CHANGELOG](../CHANGELOG.md) in the `release/X.Y.Z.A.n` branch.
```

reason being that we should all be updating the changelog in the PR that merges the code and not backfilling it if possible.

Reply:
I think we should always update the changelog here, because we need to change the `Unreleased` header to the release number. We should clarify that point here.

Reply:
fair point - i think i'm getting ahead of myself here. i'll resolve